lib/gen_stage.ex

```diff
@@ -974,6 +975,15 @@ defmodule GenStage do
               | {:stop, reason, new_state}
             when new_state: term, reason: term
 
+  @doc """
+  Invoked when items are discarded from the buffer.
+
+  It receives the number of excess (discarded) items from this invocation.
+  This callback returns a boolean that controls whether the default error log for discarded items is printed or not.
+  Return true to print the log, return false to skip the log.
```

Comment:
```suggestion
  This callback returns a boolean that controls whether the default error
  log for discarded items is printed or not. It should return `true` to print
  the log, `false` otherwise.
```